{{cookiecutter.project_slug}}/README.md

```diff
@@ -10,6 +10,9 @@
 
 ## Features
 
+- This package uses [pydocstyle](http://www.pydocstyle.org/en/stable/)
```

Comment:
you will need to check first if the user selected pydocstyle.
do it in a new PR.